core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/kubernetesModels.scala

```diff
@@ -439,9 +439,9 @@ final case class App(id: AppId,
                      extraArgs: List[String],
                      sourceWorkspaceId: Option[WorkspaceId],
                      numOfReplicas: Option[Int],
-                     autodeleteEnabled: Boolean,
-                     autodeleteThreshold: Option[AutodeleteThreshold],
-                     autopilot: Option[Autopilot]
+                     autodelete: Autodelete,
+                     autopilot: Option[Autopilot],
+                     mountWorkspaceBucketName: Option[String]
```

Comment:
woulld be good to use `GcsBucketName`

Reply:
I want to keep `mount` in the name to indicate that this bucket will be used in mount. If this field is not resuable for Azure k8s app, I can rename to `mountGcsBucketName`? 

@cindy-broadinstitute wdyt?

Reply:
I think that `mount` being a verb is confusing in the name of the bucket. How about externalBucketName?

Reply:
Or some other name indicating it is not the Workspace Storage Bucket, but is another bucket outside the Workspace that we are making available in the Workspace?

Reply:
@yonghaoy I meant the type of the variable...so like `mountWorkspaceBucketName: Option[GcsBucketName]`

Reply:
I wish we didn't have cloud resources named for a particular cloud instead of generic, but since we do, I agree with @Qi77Qi to set the type.

Reply:
this bucket can be workspace bucket and external bucket, as long as user has permission to that bucket. In our initial use case, it will be ws bucket. 
`mount being a verb is confusing` I might go with 
`bucketNameToMount: Option[GcsBucketName]`